elastic/apm-server
# beater/middleware: log event.duration

## Motivation/summary

Add duration of requests to log records, to assist in debugging performance issues when server tracing is disabled, or for unsampled requests.

## Checklist


I have considered changes for:
~- [ ] documentation~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes

1. apm-server -e
2. send events
3. observe logs have an `event.duration` field

## Related issues

None.
